radar-upload-frontend/src/components/Login/index.vue

```diff
@@ -8,48 +8,27 @@
       sm="8"
       md="4"
     >
-      <v-card class="elevation-12">
-        <v-toolbar
-          color="primary"
-          dark
-          flat
-        >
-          <v-toolbar-title>Login form</v-toolbar-title>
-        </v-toolbar>
-        <v-card-text>
-          <v-form :disabled="loading">
-            <v-text-field
-              label="Login"
-              name="login"
-              prepend-icon="mdi-account"
-              type="text"
-            />
-
-            <v-text-field
-              id="password"
-              label="Password"
-              name="password"
-              prepend-icon="mdi-lock"
-              type="password"
-            />
-          </v-form>
-        </v-card-text>
-        <v-card-actions>
-          <div class="flex-grow-1 text-align-center" />
-          <div
-            class="title pr-2"
-            style="color: grey"
-          >
-            OR
-          </div>
-          <v-btn
-            :loading="loading"
-            color="primary"
-            @click="redirectLogin"
+      <v-card
+        class="elevation-12"
+        max-height="200"
+      >
+        <v-card-title>
+          Login
+        </v-card-title>
+        <v-divider />
+        <v-row>
+          <v-col
+            align="center"
           >
-            Login by management portal
-          </v-btn>
-        </v-card-actions>
+            <v-btn
+              :loading="loading"
+              color="primary"
+              @click="redirectLogin"
+            >
+              With Management Portal
```

Comment:
Can this just redirect on page load instead of requiring user interaction? Or redirect by making the router redirect to managementportal?

Reply:
It could be, but in case token expires, automatic redirecting to another website may cause confusing